### ISSUE TITLE
Exception handling bug in tasks controller

### ISSUE THREAD

**Issue by roscisz** [roscisz/TensorHive#294]
```
Traceback (most recent call last):
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/connexion/decorators/decorator.py", line 48, in wrapper
    response = function(request)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/connexion/decorators/security.py", line 299, in wrapper
    return function(request)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/connexion/decorators/uri_parsing.py", line 143, in wrapper
    response = function(request)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/connexion/decorators/validation.py", line 347, in wrapper
    return function(request)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/connexion/decorators/parameter.py", line 126, in wrapper
    return function(**kwargs)
  File "/home/tensorhive/TensorHive/venv/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 103, in wrapper
    return fn(*args, **kwargs)
  File "/home/tensorhive/TensorHive/tensorhive/controllers/task.py", line 172, in get_all
    return content, status
UnboundLocalError: local variable 'content' referenced before assignment
```

**Comment by jvdavim** [comment]
I got this error when trying to login Web App. Am I doing something wrong?

**Comment by srandoux** [comment]
need to use good pyJWT version, pip install PyJWT==1.7.1

**Comment by roscisz** [comment]
@jvdavim sorry for the late response, for some reason I didn't see the notification about your comment.

This bug here is a generic one, however, the solution suggested by @srandoux would most probably be the right one for you.

Both bugs are fixed in #337, which we plan to release in a new version within weeks.

**Comment by roscisz** [comment]
fixed in #337
